assets/components/src/plugin-settings/SettingsSection.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Ads Settings Section.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ActionCard, Grid, Button, TextControl, CheckboxControl, SelectControl } from '../';
+
+const SettingsSection = ( { settings, disabled, onChange, onUpdate } ) => {
```

Comment:
There's already a [`Settings.SettingsSection` component](https://github.com/Automattic/newspack-plugin/blob/47237490b34e190d744b2333b7483e2805eb7c84/assets/components/src/settings/SettingsSection.js) in this repository, but it seems unused in favour of `SettingsCard`. _If_ this component was less tied to the plugin settings API (see the other comment below), it could replace the `Settings.SettingsSection` component, so it could be used more widely.

Reply:
The `PluginSettings.Section` component is more abstract now, since fb2b80d04188e0287f7e8e31e104de485ecd27e9.

I believe we can rearrange/deprecate components in a different PR. WDYT?

Reply:
Better to do it here IMO, but it's non-blocking - so as you prefer 👌

Reply:
I think there's already a lot going on in this PR 😅 
I'd rather have it as another PR, in case further discussion is required we'll have clean space for it.